src/Microsoft.Net.Http.Headers/SetCookieHeaderValue.cs

```diff
@@ -342,15 +341,15 @@ private static int GetSetCookieLength(string input, int startIndex, out SetCooki
                     }
                     var dateString = ReadToSemicolonOrEnd(input, ref offset);
                     DateTimeOffset expirationDate;
-                    if (!HttpRuleParser.TryStringToDate(dateString, out expirationDate))
+                    if (!HttpRuleParser.TryStringToDate(dateString.ToString(), out expirationDate))
```

Comment:
It's just a wrapper for DateTimeOffset.TryParseExact and I didn't want to hide the ToString.